packages/manager/apps/dedicated/client/app/dedicatedCloud/details/dedicatedCloud.routing.js

```diff
@@ -240,6 +240,9 @@ export default /* @ngInject */ ($stateProvider) => {
             );
             return options;
           }),
+      hasNsxOption: /* @ngInject */ (optionsAvailable) =>
+        optionsAvailable.filter((option) => ['nsx', 'nsxt'].includes(option))
```

Comment:
```suggestion
        !!optionsAvailable.find((option) => ['nsx', 'nsxt'].includes(option))
```
filter creates a new array which is not really necessary here